code/modules/client/preference/character.dm

```diff
@@ -476,11 +481,12 @@
 	h_grad_offset_x = query.item[52] // parsed down below
 	h_grad_colour = query.item[53]
 	h_grad_alpha = query.item[54]
-	var/custom_emotes_tmp = query.item[55]
-	physique = query.item[56]
-	height = query.item[57]
-	cyborg_brain_type = query.item[58]
-	body_type = query.item[59]
+	runechat_color = query.item[55]
+	var/custom_emotes_tmp = query.item[56]
+	physique = query.item[57]
+	height = query.item[58]
+	cyborg_brain_type = query.item[59]
+	body_type = query.item[60]
```

Comment:
You need to also edit `20-load_characters.dm` to actually have it read from this query.

Please test this code with a database properly.

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/60483458/3ad29d85-0113-46a7-be9a-2f71d1389f89)

Tested properly, pushing a commit.

Reply:
`code/__DEFINES/misc_defines.dm` and the configuration `toml` updated.

Reply:
Here is the query with some characters with differing runechat colors @AffectedArc07 
![image](https://github.com/ParadiseSS13/Paradise/assets/60483458/7a5e3b5e-fdf7-4950-895d-25a47961a766)

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/60483458/f2f1ca40-016d-4d8b-82a5-884e5cf68efb)